source/csharp/Camera/Basic/CaptureViaZivid/CaptureViaZivid.cs

```diff
@@ -73,17 +74,28 @@ private static HalconDotNet.HObjectModel3D ZividToHalconPointCloud(Zivid.NET.Poi
         // tupleXYZMapping is of shape [width, height, rows[], cols[]], and is used for creating xyz mapping.
         // See more at: https://www.mvtec.com/doc/halcon/13/en/set_object_model_3d_attrib.html
 
-        var tupleX = new HalconDotNet.HTuple();
-        var tupleY = new HalconDotNet.HTuple();
-        var tupleZ = new HalconDotNet.HTuple();
+        var tuplePointsX = new HalconDotNet.HTuple();
+        var tuplePointsY = new HalconDotNet.HTuple();
+        var tuplePointsZ = new HalconDotNet.HTuple();
+
+        var tupleNormalsX = new HalconDotNet.HTuple();
+        var tupleNormalsY = new HalconDotNet.HTuple();
+        var tupleNormalsZ = new HalconDotNet.HTuple();
+
         var tupleR = new HalconDotNet.HTuple();
         var tupleG = new HalconDotNet.HTuple();
         var tupleB = new HalconDotNet.HTuple();
```

Comment:
Maybe you could call this tupleColorsR etc. for consistency?

Reply:
i agree, done